nupic/frameworks/opf/clamodel.py

```diff
@@ -726,15 +717,15 @@ def _handleCLAClassifierMultiStep(self, patternNZ,
         encoderList = sensor.getSelf().disabledEncoder.getEncoderList()
       else:
         encoderList = []
-      if len(encoderList) >= 1:
+
+      if encoderList: # not empty
```

Comment:
Remove the comment. If the code isn't clear enough then change it back to checking the length.